.circleci/config.yml

```
@@ -0,0 +1,61 @@
+version: 2.1
+
+filters_always: &filters_always
+  filters:
+    tags:
+      only: /.*/
+
+filters_publish: &filters_publish
+  filters:
+    tags:
+      only: /^v[0-9].*/
+    branches:
+      ignore: /.*/
+
+orbs:
+  bats: circleci/bats@1.0.0
+
+jobs:
+  smoke_test:
+    machine:
+      image: ubuntu-2004:2023.04.2
```

Updated to the newer LTS image

```suggestion
      image: ubuntu-2204:2024.01.1
```